Solutions/Palo Alto - XDR (Cortex)/Detection Queries/High Severity Cortex Incidents. yaml

```diff
@@ -0,0 +1,33 @@
+id: be8325a4-530c-484d-b301-e01be23395e2
+name: Palo Alto XDR native Incidents
+description: |
+  'The query checks for all High Log Severity Alerts and incidents fired in Cortex and returns information of enetities.'
+severity: High
+requiredDataConnectors:
+  - connectorId: CommonSecurityLog
+    dataTypes:
+      - CommonSecurityLog
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - DefenseEvasion
+relevantTechniques:
+  - T1562
+query: |
+  CommonSecurityLog
+  | where DeviceVendor == "Palo Alto Networks"
+  | where LogSeverity has_any (2,3,4,5,6)
+  | summarize by TimeGenerated,DeviceProduct, DeviceEventClassID, LogSeverity, Activity, DeviceAction, Computer, DeviceVersion, FileHash, FileName, FilePath, RequestURL, SourceHostName, SourceUserName
```

Comment:
DeviceProduct is not needed here for the summarization.
Also, do we really need the summarization here? In case we want to raise Incident for all High Severity Alerts, we do not require summarization. Please review the logic to be very sure.